oracle/tests/test_custom_queries.py

```diff
@@ -107,3 +107,29 @@ def test_custom_queries_multiple_results(aggregator, check):
     aggregator.assert_metric(
         "oracle.test1.metric", value=2, count=1, tags=["tag_name:tag_value2", "query_tags1", "custom_tag"]
     )
+
+
+def test_custom_queries_metric_prefix_skip_column(aggregator, check, dd_run_check, instance):
+    con = mock.MagicMock()
+    cursor = mock.MagicMock()
+    data = [[["tag_value1", "1"]], [[1, 2, "tag_value2"]]]
+    cursor.fetchall.side_effect = lambda: iter(data.pop(0))
+    con.cursor.return_value = cursor
+
+    custom_queries = [
+        {
+            "metric_prefix": "oracle.test1",
+            "query": "mocked",
+            "columns": [{}, {"name": "metric", "type": "gauge"}],  # skip `tag_value1` column
+            "tags": ["query_tags1"],
+        },
+    ]
+    instance['custom_queries'] = custom_queries
+    instance['only_custom_queries'] = True
+    check = Oracle(CHECK_NAME, {}, [instance])
+
+    with mock.patch("datadog_checks.oracle.oracle.Oracle._connection", new_callable=mock.PropertyMock) as connection:
+        connection.return_value = con
+        dd_run_check(check)
+
+    aggregator.assert_metric("oracle.test1.metric", value=1, count=1, tags=["query_tags1", "optional:tag1"])
```

Comment:
I don't understand where each of this values is coming from.
`metric` is coming from `{"name": "metric"`, but where is `test1` coming from? and `optional:tag1`?

Reply:
It comes from the `instance` fixture [here](https://github.com/DataDog/integrations-core/blob/a5ea514939c855f18da007dc9771e95f5b46bd8c/oracle/tests/conftest.py#L81)